### DISCUSSION TITLE
replit

### DISCUSSION THREAD

**Discussion by Justin10407** [TannerGabriel/discord-bot#97] [Q&A]
is it possible to add it to replit? it keeps showing me errors everytime i try like 
Error: ENOENT: no such file or directory, open '/home/runner/otegbot/index.js'

**Comment by TannerGabriel** [comment]
I have personally never used `repl.it` and therefore cannot say much about it. If you can install tools like FFMPEG or run Docker containers, you should be able to run the bot, but I'm not sure if that is possible.
